app/styles/modules/_talent-report.scss

```diff
@@ -0,0 +1,29 @@
+.talent-report {
+
+  table th {
+      font-size: 11px;
+  }
+
+  table td {
+      font-size: 10px;
+  }
+
+  table td.rate5 {
+    color: rgb(203,0,3);
```

Comment:
we can use the variables that we already have in _settings.scss, i'll also show how you can do this the sass way in the next pr :)